### ISSUE TITLE
Bug: use-layers treats layer() in @import as case-sensitive

### ISSUE THREAD

**Issue by KumJungMin** [eslint/css#536]
### Environment

ESLint version: 10.8.0
@eslint/css version: 1.4.0
Node version: 22.18.0
npm version: 10.9.3
Operating System: Darwin 24.6.0

### Which language are you using?

stylesheet

### What did you do?

I enabled the css/use-layers rule and used different casing variations of the layer() function name within @import.

```js
import { defineConfig } from "eslint/config";
import css from "@eslint/css";


export default defineConfig([
    {
        files: ["**/*.css"],
        language: "css/css",
        plugins: { css },
        rules: {
            "css/use-layers": "error",
        },
    },
]);
```
```css
/* Valid: lowercase layer() */
@import "lowercase.css" layer(lowercase);


/* Should be valid, but currently produces an error */
@import "uppercase.css" LAYER(uppercase);
@import "mixed.css" Layer(mixed);
@import "mixed-case.css" lAyEr(mixed-case);
```
I ran ESLint using the following command:
```
npx eslint test.css
```

### What did you expect to happen?

I expected all @import declarations to pass without errors.

CSS Cascade Level 5 defines the optional @import syntax as follows:
```
[ layer | layer(<layer-name>) ]?
```
Here, `layer(<layer-name>)` is a function. According to CSS Values and Units Level 4, CSS function names are ASCII case-insensitive.

Therefore, the following function names should all be handled in the same way:
```css
layer(lowercase)
LAYER(uppercase)
Layer(mixed)
lAyEr(mixed-case)
```
The <layer-name> argument passed to the function is separate from the function name itself. This issue concerns the case sensitivity of the layer() function name, not the layer name.

References:

[CSS Cascade Level 5: @import](https://drafts.csswg.org/css-cascade-5/#at-import)
[CSS Values and Units Level 4: Functional Notations](https://drafts.csswg.org/css-values-4/#functional-notations)

### What actually happened?

Only the lowercase layer() function is recognized as a layer declaration.

The uppercase and mixed-case forms—`LAYER()`, `Layer()`, and `lAyEr()`—are not recognized as layer declarations, 
so `css/use-layers` reports the following error:
```
Expected import to be within a layer.

5:1  error  Expected import to be within a layer  css/use-layers
6:1  error  Expected import to be within a layer  css/use-layers
7:1  error  Expected import to be within a layer  css/use-layers
```

### Link to Minimal Reproducible Example

https://stackblitz.com/edit/vitejs-vite-bh58g65u?file=eslint.config.js,test.css

### Participation

- [x] I am willing to submit a pull request for this issue.

### AI acknowledgment

- [ ] I did not use AI to generate this issue report.
- [x] (If the above is not checked) I have reviewed the AI-generated content before submitting.

### Additional comments

> Disclosure: I'm a participant in the [OSSCA open source contribution program](https://github.com/eslint-ossca).

A related case was addressed in PR #233, which made at-rule name matching case-insensitive across several rules.

That change applies to at-rule names such as `@IMPORT` and `@LAYER`. However, the `layer()` function inside `@import` is handled by a separate comparison in `use-layers`, where its name appears to be compared directly with the lowercase string `"layer"`. As a result, uppercase and mixed-case forms of the function are not recognized.

**Comment by DMartens** [comment]
The case-insensitivity for the remaining rules (including `use-layers`) is currently being done in #530.
